core/query/annotated.go

```diff
@@ -34,7 +34,7 @@ type AnnotatedInput struct {
 	Amount          uint64             `json:"amount"`
 	IssuanceProgram chainjson.HexBytes `json:"issuance_program,omitempty"`
 	ControlProgram  chainjson.HexBytes `json:"control_program,omitempty"`
-	SpentOutput     *SpentOutput       `json:"spent_output,omitempty"`
```

Comment:
@jeffomatic @jbowens Oh, I forgot something. We no longer can get this data easily from the TxInput itself (w/o extra roundtrip to the DB), but I also have a feeling no one is actually using it because we do not have `/list-transaction-outputs`, we only have `/list-unspent-outputs` and for those we keep necessary data in the API results to query both via `transaction_id;position` and `output_id`.